demo/date-picker-validation-demos.html

```diff
@@ -6,67 +6,64 @@
       }
     </style>
 
-    <h3>Using as a Form Field</h3>
-    <vaadin-demo-snippet id="date-picker-validation-demos-using-as-a-form-field" when-defined="vaadin-date-picker">
+    <h3>Required</h3>
+    <vaadin-demo-snippet id="date-picker-required">
       <template preserve-content>
-        <iron-form>
-          <form method="post">
-            <vaadin-date-picker name="date" required error-message="This field is required"></vaadin-date-picker>
-            <vaadin-button>Submit</vaadin-button>
-          </form>
-        </iron-form>
-
+        <vaadin-date-picker label="Required" required error-message="Please choose a date" id="datePicker">
+        </vaadin-date-picker>
+        <vaadin-button>Submit</vaadin-button>
         <script>
-          window.addDemoReadyListener('#date-picker-validation-demos-using-as-a-form-field', function(document) {
-            var form = document.querySelector('iron-form');
-
-            form.addEventListener('iron-form-presubmit', function(e) {
-              // prevent actual form submission
-              e.preventDefault();
-              alert('Form submitted with date: ' + form.serializeForm().date);
-              return false;
-            });
-
-            var button = document.querySelector('vaadin-button');
-            button.addEventListener('click', function() {
-              form.submit();
+          window.addDemoReadyListener('#date-picker-required', function(document) {
+            const submit = document.querySelector('vaadin-button');
+            const datePicker = document.querySelector('vaadin-date-picker');
+            submit.addEventListener('click', function(event) {
+              datePicker.validate();
             });
           });
         </script>
       </template>
     </vaadin-demo-snippet>
 
+    <h3>Min and max date validation</h3>
+    <vaadin-demo-snippet>
+      <template preserve-content>
+        <vaadin-date-picker label="Label" min="2019-11-10" max="2019-11-11" initial-position="2020-11-11"></vaadin-date-picker>
```

Comment:
Should `initial-position` be `2019-11-11` instead?

Reply:
Done.